client/src/store/store.js

```diff
@@ -0,0 +1,48 @@
+import React, { createContext, useContext, useReducer } from 'react';
```

Comment:
from react-bootstrap's documentation for [importing components](https://react-bootstrap.github.io/getting-started/introduction#importing-components):

> You should import individual components like: react-bootstrap/Button rather than the entire library. Doing so pulls in only the specific components that you use, which can significantly reduce the amount of code you end up sending to the client.

Reply:
my bad. my comment does not apply here, please disregard.